spec/asyncapi.md

```diff
@@ -1401,6 +1490,64 @@ description: Find more info here
 url: https://example.com
 ```
 
+
+#### <a name="metadataObject"></a>Metadata Object
+
+The object provides metadata about the described object. May contain previously non defined fields.
```

Comment:
I have a problem with `May contain previously non defined fields.` sentence, because I don't know if we should allow only defining extensions and not non defined field. With this possibility there will be problems with validation on tooling side, by extension we can support it, by non defined fields no.